src/model/comment.ml

```diff
@@ -39,7 +41,7 @@ type module_reference = {
     resolved during linking. *)
 
 type nestable_block_element =
-  [ `Paragraph of paragraph
+  [ `Paragraph of paragraph_style option * inline_element with_location list
```

Comment:
Here the style argument is placed first but it's last for `Heading`, for consistency I suggest picking one side and changing the other.